app/views/admin/enterprises/form/_primary_details.html.haml

```diff
@@ -70,8 +70,8 @@
       %div{'ofn-with-tip' => t('.link_to_front_tip')}
         %a= t('admin.whats_this')
     .eight.columns.omega
-      = surround main_app.root_url, "/shop" do
-        {{Enterprise.permalink}}
+      - link_to_front_shop = [main_app.root_url, "{{Enterprise.permalink}}", "/shop"].join
```

Comment:
What do you think about:

```ruby
- link_to_front_shop = "#{main_app.root_url}#{@enterprise.permalink}'/shop'"
```
It removes the use of `{{ Enterprise.permalink }}` ie. AngularJS, which we try to remove.

Reply:
Sounds good! I am glad that there is the aim to remove AngularJS. Do you want to replace it with another Frontend-Framework, use StimulusJS and/or moving to hotwire? Can you point me to the discussion about it? :)

Reply:
Yes we want to move to StimulusJS and CableReady. 

See that issue: https://github.com/openfoodfoundation/openfoodnetwork/issues/9477